### PR TITLE
Language support in Swedish and Danish for Best Practise Checkout guide

### PR DESCRIPTION
**Pull request by dipyayan**
In this PR, we introduce the language support for Best Practise Checkout guide in Swedish and Danish.
Screenshot 2023-09-18 at 22 38 48
Screenshot 2023-09-18 at 22 39 10
Screenshot 2023-09-18 at 22 39 41
Screenshot 2023-09-18 at 22 39 55

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/gCLPBYjv](https://trello.com/c/gCLPBYjv)

File: config.json [hunk 1]
```diff
@@ -14,6 +14,14 @@
     "no": {
       "staticDir": ["static"],
       "languageName": "Norsk"
+    },
+    "se": {
+      "staticDir": ["static"],
+      "languageName": "Swedish"
```
Comment (cdwinsnes) [inline config.json:20]: Svensk

File: config.json [hunk 2]
```diff
@@ -14,6 +14,14 @@
...
+    },
+    "da": {
+      "staticDir": ["static"],
+      "languageName": "Danish"
```
[the remark this answers is not in the document]
Comment (dipyayan) [reply config.json]: done

File: config.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (dipyayan) [reply config.json]: done

File: i18n/se.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "norway": "Norge",
+  "sweden": "Svenska",
```
[the remark this answers is not in the document]
Comment (dipyayan) [reply i18n/se.json]: done

**Review by esschul — APPROVED** [review]

**Pull request merged**